src/MyEventEmitter.js

```diff
@@ -1,14 +1,65 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+
+  getListener(event) {
+    if (!this.isListenerExist(event)) {
+      this.listeners[event] = [];
+    }
+
+    return this.listeners[event];
+  }
+
+  isListenerExist(event) {
+    return this.listeners[event];
+  }
+
+  on(event, callback) {
+    this.getListener(event).push({ callback });
+  }
+
+  once(event, callback) {
+    this.getListener(event).push({ callback, once: true });
+  }
+
+  off(event, callback) {
+    if (!this.isListenerExist(event)) {
+      return;
+    }
+
+    this.listeners[event] = this.listeners[event].filter(
+      (listener) => listener.callback !== callback,
```

Comment:
The comma at the end of the arrow function in the `filter` method is unnecessary and could be removed for clarity.